dockstore-integration-testing/src/test/java/io/dockstore/client/cli/BaseIT.java

```diff
@@ -350,6 +351,33 @@ protected static io.dockstore.openapi.client.ApiClient getAnonymousOpenAPIWebCli
         return client;
     }
 
+    protected static void verifyChecksumsAreSaved(List<Tag> tags) {
```

Comment:
If these are inherited, they should have javadoc
If I understand correctly, these probably came about because a test file got split into two versions, one with bitbucket and one without. It may be more clear to add an additional class. 

i.e. BaseIT is the base class for a new class that is the parent of BitbucketGeneralWorkflowIT and GeneralWorkflowIT since these methods are unlikely to be re-used elsewhere?